src/cli.rb

```diff
@@ -150,6 +150,19 @@ def run
         c.action Commands::Status
       end
 
+      command :ipmi do |c|
+        c.syntax = 'metal ipmi NODE_IDENTIFIER [options]'
+        c.summary = 'Perform ipmi commands on single or multiple machines'
+        c.description = "***VOLATILE***\n\n" \
```

Comment:
I have updated it so it says `volatile` when you run `metal --help`. The other texts I was going to keep as eventually it will be in the CLI